sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -120,6 +122,12 @@ const createHTLC = async (
     const assetExchangeAgreementStr = createAssetExchangeAgreementSerialized(assetType, assetID, recipientECert, "");
     const lockInfoStr = createAssetLockInfoSerialized(hash, expiryTimeSecs);
 
+    //If timeoutCallback is not defined, start automated listener
+    if (!timeoutCallback) 
+    {
+        StartHTLCAssetLockListener(contract, "", assetType, assetID, recipientECert, "", assetLockCallbackTimed);
```

Comment:
I'd prefer to name the callback function `assetLockExpirationCallback`. (See further below.)